[2022-01-29 12:27:32,829] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-02-04T00:00:00+00:00 [queued]>
[2022-01-29 12:27:32,902] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-02-04T00:00:00+00:00 [queued]>
[2022-01-29 12:27:32,902] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:27:32,903] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-29 12:27:32,903] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:27:33,001] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-02-04 00:00:00+00:00
[2022-01-29 12:27:33,137] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-02-04T00:00:00+00:00', '--job-id', '263', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpgp9s5qzg', '--error-file', '/tmp/tmp2ln8qg54']
[2022-01-29 12:27:33,150] {standard_task_runner.py:77} INFO - Job 263: Subtask local_to_gcs_task
[2022-01-29 12:27:33,078] {standard_task_runner.py:52} INFO - Started process 3447 to run task
[2022-01-29 12:27:33,707] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-02-04T00:00:00+00:00 [running]> on host 85e1117cfb12
[2022-01-29 12:27:34,131] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-04T00:00:00+00:00
[2022-01-29 12:27:35,851] {python.py:175} INFO - Done. Returned value was: None
[2022-01-29 12:27:35,874] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20190204T000000, start_date=20220129T122732, end_date=20220129T122735
[2022-01-29 12:27:35,997] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 12:27:36,163] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
